ui/utils.py

```diff
@@ -91,7 +91,7 @@ def query_moira_lists(user):
     moira_user = get_moira_user(user)
     moira = get_moira_client()
     try:
-        list_infos = moira.user_list_membership(moira_user.username, moira_user.type)
+        list_infos = moira.user_list_membership(moira_user.username, moira_user.type, max_return_count=100000)
```

Comment:
Does this have effect on performance of api?

Reply:
It could cause a performance hit only for users who belong to many thousands of moira lists.  Without this change, those users may be denied access to collections/videos that they should be able to view.